CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# astro changelog
+
+## 0.4.2 (UNRELEASED, 2018)
+
+* Fixed `--help` displaying "pflag: help requested" (#1)
```

Comment:
Can you please add a note here saying that if the user has configured flags incorrectly, then those flags just won't work or show up.

Reply:
OK. Actually, I'm going to investigate a more thorough fix. Will report back here.